addons/sale_pdf_quote_builder/utils.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import io
+
+from odoo import _
+from odoo.exceptions import ValidationError
+from odoo.tools import pdf
+
+
+def _ensure_document_not_encrypted(document):
+    """TODO edm (alphanum (this include underscore) or -) once or more ( at this point, we didn't replace __ by . yet, so no need to cover that case)
```

Comment:
:eyes:

Reply:
That's what happen when I lost my coffee buddy... :facepalm:

Reply:
:coffee: